orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -4927,3 +4926,54 @@ def run_step(model, x):
     _test_helpers.assert_values_are_close(pt_prediction, ort_prediction)
     _test_helpers.assert_values_are_close(pt_loss, ort_loss)
     _test_helpers.assert_values_are_close(pt_x.grad, ort_x.grad)
+
+
+@pytest.mark.parametrize("M, N", [(2400, 128), (2400, 256), (2400, 512), (2400, 1024), (2400, 2048), (2400, 4096), (2400, 12800)])
+def test_tensorrt_train_sample_model(M, N):
```

Comment:
Do you need to enable something to build ORT so TRT can work? So do we need to change the pipeline config so that this test is run using TRT EP? What if I run this UT on an ORT that doesn't built with TRT? I am guessing the options will be ignored and the UT is still be run but use CUDA only? Is it possible we detect the ORT has TRT enabled and only run this test when it's true?

Reply:
You are right. We need enable this test only on TRT enabled build (and pipeline).

Reply:
I don't think we have any ORTModule tests being run in a TRT pipeline right now. We might need to create a new pipeline for that?

Reply:
Yeah, we need add a new pipeline for that kind of tests. :)